tutorials/toolkits/axes_grid.py

```diff
@@ -1,115 +1,84 @@
-position of multiple axes at the drawing time. `ParasiteAxes`_
-provides twinx(or twiny)-like features so that you can plot different
-data (e.g., different y-scale) in a same Axes. `AnchoredArtists`_
-includes custom artists which are placed at some anchored position,
-like the legend.
+- Helper classes (ImageGrid_, RGBAxes_, AxesDivider_) to ease the layout of
+  axes displaying images with a fixed aspect ratio while satisfying additional
+  constraints (matching the heights of a colorbar and an image, or fixing the
+  padding between images);
+- ParasiteAxes_ (twinx/twiny-like features so that you can plot different data
+  (e.g., different y-scale) in a same Axes);
+- AnchoredArtists_ (custom artists which are placed at an anchored position,
+  similarly to legends).
 
 .. figure:: ../../gallery/axes_grid1/images/sphx_glr_demo_axes_grid_001.png
    :target: ../../gallery/axes_grid1/demo_axes_grid.html
    :align: center
    :scale: 50
 
-   Demo Axes Grid
-
-
 axes_grid1
 ==========
 
 ImageGrid
 ---------
 
-A grid of Axes.
-
-In Matplotlib, the axes location (and size) is specified in normalized
-figure coordinates. This may not be ideal for images that needs to be
-displayed with a given aspect ratio; for example, it is difficult to
-display multiple images of a same size with some fixed padding between
-them.  `~.axes_grid1.axes_grid.ImageGrid` can be used in such a case; see its
-docs for a detailed list of the parameters it accepts.
+In Matplotlib, axes location and size are normally specified in normalized
+figure coordinates (0 = bottom left, 1 = top right), which makes
+it difficult to achieve a fixed (absolute) padding between images.
+`~.axes_grid1.axes_grid.ImageGrid` can be used to achieve such a padding; see
+its docs its detailed API.
 
 .. figure:: ../../gallery/axes_grid1/images/sphx_glr_simple_axesgrid_001.png
    :target: ../../gallery/axes_grid1/simple_axesgrid.html
    :align: center
    :scale: 50
 
-   Simple Axesgrid
-
 * The position of each axes is determined at the drawing time (see
-  `AxesDivider`_), so that the size of the entire grid fits in the
+  AxesDivider_), so that the size of the entire grid fits in the
   given rectangle (like the aspect of axes). Note that in this example,
   the paddings between axes are fixed even if you changes the figure
   size.
 
-* axes in the same column has a same axes width (in figure
-  coordinate), and similarly, axes in the same row has a same
-  height. The widths (height) of the axes in the same row (column) are
-  scaled according to their view limits (xlim or ylim).
+* Axes in a same column share their x-axis, and axes in a same row share their
+  y-axis (in the sense of `~.Axes.sharex`, `~.Axes.sharey`).  Additionally,
+  Axes in a same column all have the same width, and axes in a same row all
+  have the same height.  These widths and heights are scaled in proportion to
+  the axes' view limits (xlim or ylim).
```

Comment:
The 'the' to 'a' change doesn't read right; either `Axes in a column` or back to `Axes in the same column`.